### PR TITLE
Added landing Page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Screenshot (207)

**Added folders for pages and components**
Converted my html landing page code into jsx 
No functionalities are added in this pr , just simple addition of landing page UI

### CODE REVIEW

File: src/WebPages/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  padding-top: 1rem;
+}
+
+.primary-button {
+  cursor: pointer;
+}
+
+/* .horizontal-card-component :nth-child(1) {
+  height: 154.125px;
+} */
```
Comment (anuj-kumary) [inline src/WebPages/LandingPage/LandingPage.css]: Remove unwanted comment

**Review by anuj-kumary — APPROVED** [review]
Loved how you set up your folder structure
Clean code gave better readability

File: src/App.js [hunk 2]
```diff
@@ -2,12 +2,18 @@ import "./App.css";
...
 function App() {
   return (
-    <Routes>
-      <Route path="/mockman" element={<Mockman />} />
-    </Routes>
+    <>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="/mockman" element={<Mockman />} />
```
Comment (Rohanmond) [inline src/App.js:14]: delete this route , you already defined in the upper

**Review by Rohanmond — APPROVED** [review]

**Review by mdarif6 — APPROVED** [review]
- Code structure is okay.
- As you got other's comment- remove comment code and 
- refactor router too.

**Pull request merged**